.projenrc.js

```diff
@@ -3,7 +3,7 @@ const { ReleaseTrigger } = require('projen/lib/release');
 const project = new awscdk.AwsCdkConstructLibrary({
   author: 'Ben Limmer',
   authorAddress: 'https://benlimmer.com/freelance',
-  cdkVersion: '2.1.0',
+  cdkVersion: '2.56.0',
```

Comment:
@hlascelles, since this project uses `projen` (https://projen.io/), you don't manually update `package.json`/`yarn`, etc. This is all managed through projen. I'm kind of surprised you were able to edit `package.json`, as projen marks those files as read-only.

So, with projen, to update anything, you handle it through this configuration file. When you make changes, you then run `yarn projen`, which updates all the dependent subfiles.

To produce this PR I:

1. Checked out your branch.
2. Discarded this commit where you updated the `package.json` [`79111cd` (#52)](https://github.com/blimmer/cdk-datadog-integration/pull/52/commits/79111cdbee9e09b76944825ff2ca48031a72799e)
3. Updated this `cdkVersion` line in this file
4. Ran `yarn projen`
5. Git add all and commit!